src/test/java/pages/MainPage.java

```diff
@@ -0,0 +1,148 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+public class MainPage {
+
+    public MainPage(WebDriver driver) {
+        PageFactory.initElements(driver, this);
+    }
+
+    @FindBy(className = "goog-imageless-button-content")
+    public WebElement calendarButton;
+
+    @FindBy(xpath = "//DIV[@class='goog-imageless-button-content'][text()='Create']")
+    public WebElement createButton;
+
+    @FindBy(id = "gridcontainer")
+    public WebElement eventsContainer;
+
+    @FindBy(className = "neb-date")
+    public WebElement dateOfTheEvent;
+
+    @FindBy(id = "currentDate:2")
+    public WebElement currentDate;
+
+    @FindBy(id = "navForward:2")
+    public WebElement nextButtonOnWeekView;
+
+    @FindBy(className = "neb-date")
+    public WebElement dateInEventPreview;
+
+    public MainPage calendarButtonIsDisplayed() {
+        Assert.assertEquals(calendarButton.isDisplayed(), true);
+        return this;
+    }
+
+    public MainPage clickOnCtreateButton() {
+        createButton.click();
+        return this;
+    }
+
+    public MainPage clickNextButtonOnWeekView() {
+        nextButtonOnWeekView.click();
+        return this;
+    }
+
+    public boolean startDateIsOnCurrentPage(String date) throws ParseException {
+        SimpleDateFormat date2 = new SimpleDateFormat("MM/dd/yyyy");
```

Comment:
I think you can use more common name instead of 'date2'  and 'date3' etc.
e.g.: date2 -> usDateFormatter